examples/kubernetes/secrets/subvars.yml

```diff
@@ -0,0 +1,5 @@
+var1:
```

Comment:
I assume this file is here just to demonstrate how the yml looks like? We shouldn't need this extra file in the final user flow right?

Reply:
Yes, this is used for demonstration and testing only. The user would store this somewhere else.

Reply:
Ok, it would also be good to add updated examples for these so that people don't have to figure out multiline string syntax etc to write secrets.

Reply:
I think a comment here will suffice

Reply:
Added comment, resolving this :)